src/routes/test/+page.svelte

```diff
@@ -2758,9 +2791,103 @@
 			</div>
 		</div>
 	</section>
+	<section class="table">
+		<Table>
+			<Tr>
+				<Th class="no-padding"
+					><Checkbox
+						class="align"
+						label="Table header"
+						{mixed}
+						bind:checked={selectionAll}
+						onclick={selectAll}
+					/><ChevronDown class="align" size={24} /></Th
+				>
+				<Th>Table header</Th>
+				<Th>Table header</Th>
+				<Th>Table header</Th>
+				<Th>Table header</Th>
+			</Tr>
+			<Tr active={selected[0]}>
+				<Td class="no-padding"
+					><Checkbox
+						class="align"
+						label="Cornhole"
+						bind:checked={selected[0]}
+						onclick={() => toggleSelection(0)}
+					/></Td
+				>
+				<Td>Glossier</Td>
+				<Td>Skateboard</Td>
+				<Td>Paleo</Td>
+				<Td>Fingerstache</Td>
+			</Tr>
+			<Tr active={selected[1]}>
+				<Td class="no-padding"
+					><Checkbox
+						class="align"
+						label="Cornhole"
+						bind:checked={selected[1]}
+						onclick={() => toggleSelection(1)}
+					/></Td
+				>
+				<Td>Glossier</Td>
+				<Td>Skateboard</Td>
+				<Td>Paleo</Td>
+				<Td>Fingerstache</Td>
+			</Tr>
+		</Table>
+	</section>
+	<section class="table">
+		<Table>
+			<Tr>
+				<Th><div class="space-between">Table header<ChevronDown class="align" size={24} /></div></Th
+				>
+				<Th>Simple text</Th>
+				<Th>Longer text</Th>
+				<Th>Badge</Th>
+				<Th style="text-align: right;">Number</Th>
+				<Th style="text-align: right;">Button</Th>
+			</Tr>
+			<Tr>
+				<Td>Cornhole</Td>
+				<Td>Glossier</Td>
+				<Td>Copper mug skateboard fingerstache</Td>
+				<Td><Badge dimension="small"><CheckmarkFilled size={16} />Small Badge</Badge></Td>
+				<Td style="text-align: right;">999 EUR</Td>
+				<Td style="text-align: right;"
+					><Button variant="secondary" dimension="compact">Outline button</Button></Td
+				>
+			</Tr>
+			<Tr>
+				<Td>Cornhole</Td>
+				<Td>Glossier</Td>
+				<Td>Copper mug skateboard fingerstache</Td>
+				<Td><Badge dimension="small"><CheckmarkFilled size={16} />Small Badge</Badge></Td>
+				<Td style="text-align: right;">999 EUR</Td>
+				<Td style="text-align: right;"
+					><Button variant="secondary" dimension="compact">Outline button</Button></Td
+				>
+			</Tr>
+		</Table>
+	</section>
 </div>
 
 <style>
+	.table {
+		margin-bottom: 5rem;
+	}
+	:global(.no-padding) {
+		padding: 0 !important;
+	}
```

Comment:
This is a big no-no. Never use `!important` especially in UI library.

Here specifically it would be better to add a prop `no-padding`.